content/governance.md

```diff
@@ -145,6 +147,34 @@ A vote on a single proposal is considered successful if at least two thirds of t
 
 If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
 
+## On- / Offboarding
+
+Changes in the On- /Offboarding section can be taken by [lazy consensus](#consensus) unless challenged.  If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Onboarding
+
+The new member is
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this thread, acknowleding team membership.
+* added to the [GitHub organization][gh] as _Owner_.
+* optinally added to the community, junkyard, and related repositories.
+* added to the [team mailing list][team].
+* announced to CNCF.
+* given access to the shared password storage.
+
+### Offboarding
+
+The ex-member is
+* removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
+* removed from the [GitHub organization][gh] and related organizations and repositories.
+* removed from the [team mailing list][team].
+* announced as removed to CNCF. We will explicitly request reconfirmation of removal from CNCF.
+* removed from shared password storage. All passwords, API tokens, etc are cycled.
+* not allowed to call themselves an active team member any more, nor allowed to imply this to be the case.
+* added to an alumni list if they agree to it.
+
+If needed, we reserve the right to publicly announce removal.
+
 ## FAQ
 
 This section is informational. In case of disagreement, the rules above overrule any FAQ.
```

Comment:
Now that we have a section that is marked explicitly as changeable by lazy consensus (or a normal vote if needed), perhaps we could do the same here? Or just cut the whole doc into the actual governance and and informational appendix (with onboarding/offboarding list, FAQ, and possibly more).

Reply:
Good point; I will add that. For now, I would prefer one doc, but no strong opinion.

Reply:
I phrased it in an unclear way. I didn't want to create two documents, but have a separator in the one doc that says something like "everything from here in is informational and can be changed by lazy consensus etc."